app/scripts/controllers/app-state.js

```diff
@@ -288,10 +288,12 @@ export default class AppStateController extends EventEmitter {
       return;
     }
 
+    const timeoutToSet = Number(timeoutMinutes);
```

Comment:
It looks like we're converting this to a number here. But the changes elsewhere in the PR suggest that it should now be expected to be a number already. Could you clarify what type we'd expect this to be?

Reply:
I'm guessing (based on some further discussion in Slack) that this value could be a number or a string, because we haven't yet written a migration to handle the malformed string values that were persisted in state.

We should add a comment explaining that this is a temporary solution until we can implement a migration, assuming that is correct

Reply:
comment added in c6614f86cc